enterprise/backend/test/metabase_enterprise/serialization/test_util.clj

```diff
@@ -330,8 +330,7 @@
                    PulseCard           [{~'pulsecard-root-id :id} {:pulse_id ~'pulse-id
                                                                    :card_id  ~'card-id-root}]
                    PulseCard           [{~'pulsecard-collection-id :id} {:pulse_id ~'pulse-id
-                                                                         :card_id  ~'card-id}
-                                                         :query {:source-table (str "card__" ~'card-id-root)}]
+                                                                         :card_id  ~'card-id}]
```

Comment:
Toucan 2's version of Toucan 1 is actually a little better at validation and caught this -- this actually has 4 args instead of the usual one `[bindings]` or 2 `[bindings properties]`. In Toucan 1 those extra two args would be completely ignored. I went ahead and deleted them, and it didn't affect anything.